pages/give-devices.js

```diff
@@ -9,21 +9,19 @@ import SelectFormControl from '../components/form/SelectFormControl';
 import TextareaFormControl from '../components/form/TextareaFormControl';
 
 // Use ISO 3166 country codes per https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes
-const COUNTRIES = [
-  { value: 'US', label: 'United States' },
-  { value: 'CA', label: 'Canada' },
-  { value: 'MX', label: 'Mexico' },
-  { value: 'DE', label: 'Germany' },
-  { value: 'LV', label: 'Latvia' },
-  { value: 'IN', label: 'India' },
-  { value: 'BD', label: 'Bangladesh' },
-  { value: 'PH', label: 'Philippines' },
-];
+const countryList = require('country-list');
+
+const countries = countryList.getData();
+countries.sort((a, b) => (a.name > b.name && 1) || -1);
+const COUNTRIES = [];
+countries.forEach((element) => {
+  COUNTRIES.push({ value: element.code, label: element.name });
+});
 
 const formDataInitialState = {
   name: '',
   email: '',
-  country: COUNTRIES[0].value,
+  country: COUNTRIES[235].value,
```

Comment:
With the changes I have suggested at +14 to +19 this line would be replaced with

```suggestion
  country: COUNTRIES[235].label,
```